02-peer-protocol.md

```diff
@@ -1235,6 +1260,17 @@ acceptance of the new channel.
    2. data:
         * [`...*byte`:`type`]
    1. type: 2 (`require_confirmed_inputs`)
+   1. type: 3 (`will_fund`)
+   2. data:
+       * [`signature`:`signature`]
+       * [`lease_rates`:`lease_rates`]
+
+1. subtype: `lease_rates`
+2. data:
+       * [`u16`:`funding_weight`]
+       * [`u16`:`lease_fee_basis`]
+       * [`u16`:`channel_fee_basis_max`]
+       * [`u32`:`lease_fee_base_sat`]
```

Comment:
We can keep channel base fees in a way where we can omit it thanks to truncated ints:

```suggestion
       * [`u32`:`lease_fee_base_sat`]
       * [`tu32`:`channel_fee_base_msat_max`]
```